### ISSUE TITLE
Spawning a new instance does not retain the --config-file command-line option 

### ISSUE THREAD

**Issue by marioroy** [alacritty/alacritty#4187]
Thank you for the `--config-file` command-line option. Unfortunately, spawning a new window instance via the keyboard shortcut `SpawnNewInstance` loads the default configuration and not the configuration specified for the initial window.

My default configuration `%APPDATA%/alacritty/alacritty.yml` is configured for WSL 2 Ubuntu 20.04.
```text
window:
  dynamic_title: false
  title: Ubuntu

# WSL 2 Ubuntu 20.04
shell:
  program: C:\Windows\System32\bash
  args:
    - --login

working_directory: C:\Users\mario
```

I have another config for Cygwin `%APPDATA%/alacritty/alacritty_cygwin.yml`.
```text
window:
  dynamic_title: false
  title: Cygwin

shell:
  program: C:\cygwin64\bin\bash
  args:
    - --login

working_directory: C:\cygwin64\home\mario
```

I have two desktop icons `alacritty - ubuntu` and `alacritty - cygwin`. The one for Cygwin is called with the option `--config-file %APPDATA%\alacritty\alacritty_cygwin.yml`. Spawning a new instance window does not check if the config-file option was specified initially so that to use the same config when spawning a new window.

Regards.

**Comment by chrisduerr** [comment]
I think it makes sense to persist CLI arguments. It should also be fairly easy to do so.

**Comment by marioroy** [comment]
Today, I learned of being able to specify a command and arguments. Thank you.
```text
key_bindings:
  - { key: K,        mods: Command, mode: ~Vi, chars: "\x0c" }
  - { key: L,        mods: Command, mode: ~Vi, chars: "\x0c" }
  - { key: C,        mods: Command, action: Copy             }
  - { key: V,        mods: Command, action: Paste            }
  - { key: Q,        mods: Command, action: Quit             }
  - { key: W,        mods: Command, action: Quit             }
  - { key: N,        mods: Command, command: { program: 'C:\Users\mario\Downloads\Alacritty-v0.5.0-windows-portable\alacritty.exe', args: ['--config-file', 'C:\Users\mario\AppData\Roaming\alacritty\alacritty_cygwin.yml'] } }
  - { key: Key0,     mods: Command, action: ResetFontSize    }
  - { key: Equals,   mods: Command, action: IncreaseFontSize }
  - { key: Add,      mods: Command, action: IncreaseFontSize }
  - { key: Subtract, mods: Command, action: DecreaseFontSize }
  - { key: Minus,    mods: Command, action: DecreaseFontSize }
  - { key: K,        mods: Command, action: ClearHistory     }
  - { key: L,        mods: Command, action: ClearLogNotice   }
```

**Comment by kchibisov** [comment]
Fixed by #4197.
